pom.xml

```diff
@@ -107,6 +107,12 @@
       <version>1.6</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-multibranch</artifactId>
```

Comment:
Nope nope nope. We are not adding this. It has the potential to create circular dependency when using local snapshots. Just dual-purpose a top level item as scmsourceowner for tests